eng/pipelines/checkout-windows-task.yml

```diff
@@ -5,10 +5,12 @@ steps:
   - checkout: none
 
   - script: |
-      @echo on
       set PATH=$(Agent.HomeDirectory)\externals\git\cmd;%PATH%
+      @echo on
       git init
+      git config --local checkout.workers 0
```

Comment:
I thought just like a week or so ago we had to flip to an AzDO task for `git` checkout as `git` was removed from the images. Did something change that put `git` back on the machines?

Reply:
Nice. Glad we found a way to keep using our `git` customizations